Wikipedia/Code/WMFModalImageGalleryViewController.m

```diff
@@ -478,6 +525,13 @@ - (void)updateImageForCell:(WMFImageGalleryCollectionViewCell*)cell
         [self setPlaceholderImage:download.image ofInfo:infoForImage forCellAtIndexPath:indexPath];
     }]
     .catch(^(NSError* error) {
+        @strongify(self);
+        BOOL const isInvalidURL =
```

Comment:
So we are frequently attempting to load invalid urls here? What's the cause of that?